backend/src/models/comment.js

```diff
@@ -15,8 +15,7 @@ class Comment {
         const [key, value] = Object.entries(queryObject)[0]
         return this.db
             .insert('comment', { [key]: value, body, userID })
-            /* Still not sure on this, seems wasteful to send all new data */
-            .then((id) => ({ body, userID, id, timestamp: Date.now() }))
+            .then((id) => this.getComment(id))
```

Comment:
Ahh because otherwise we didn't get all fields, we just got the id.

Reply:
Yes this was actually broken. Only reason it didn't break frontend was because of another bug I think